test/rules/number-literal-format/test.ts.lint

```diff
@@ -3,38 +3,41 @@
 10;
 1.1e10;
 
-01;
-~~ [leading-0]
+-0000006000;
+ ~~~~~~~~~~ [leading-0]
 
-1.
-~~ [trailing-decimal]
+-777.;
+ ~~~~ [trailing-decimal]
 
-0.0;
-~~~ [trailing-0]
-0.50;
-~~~~ [trailing-0]
+-0.000;
+ ~~~~~ [trailing-0]
 
-.5;
-~~ [leading-decimal]
+-0.90000;
+ ~~~~~~~ [trailing-0]
 
-.50;
-~~~ [trailing-0]
-~~~ [leading-decimal]
+-.2;
+ ~~ [leading-decimal]
 
-1e01;
-  ~~ [leading-0]
-1E01;
```

Comment:
Is there a reason you're removing valid syntax tests such as `1e01;` and `1E01;`? In the case of folks not using Prettier, we'd still want to validate these cases working.

Reply:
Oops. Returned that tests back